1-js/01-getting-started/2-manuals-specifications/article.md

```diff
@@ -1,42 +1,42 @@
 
-# Manuals and specifications
+# मैनुअल और विनिर्देशों
 
-This book is a *tutorial*. It aims to help you gradually learn the language. But once you're familiar with the basics, you'll need other sources.
+यह पुस्तक एक * ट्यूटोरियल * है। इसका उद्देश्य आपको धीरे-धीरे भाषा सीखने में मदद करना है। एक बार जब आप मूल बातें से परिचित हो जाते हैं, तो आपको अन्य स्रोतों की आवश्यकता होगी।
 
-## Specification
+## विशिष्टता
 
-[The ECMA-262 specification](https://www.ecma-international.org/publications/standards/Ecma-262.htm) contains the most in-depth, detailed and formalized information about JavaScript. It defines the language.
+[ECMA-262 विनिर्देशन] (https://www.ecma-international.org/publications/standards/Ecma-262.htm) में {जावास्क्रिप्ट} के बारे में सबसे अधिक गहन, विस्तृत और औपचारिक जानकारी है। यह भाषा को परिभाषित करता है।
 
-But being that formalized, it's difficult to understand at first. So if you need the most trustworthy source of information about the language details, the specification is the right place. But it's not for everyday use.
+लेकिन औपचारिक होने के नाते, यह पहली बार में समझना मुश्किल है। इसलिए यदि आपको भाषा के विवरण के बारे में सबसे भरोसेमंद स्रोत की आवश्यकता है, तो विनिर्देश सही जगह है। लेकिन यह रोजमर्रा के उपयोग के लिए नहीं है।
 
-A new specification version is released every year. In-between these releases, the latest specification draft is at <https://tc39.es/ecma262/>.
+हर साल एक नया विनिर्देश संस्करण जारी किया जाता है। इन रिलीज़ों के बीच, नवीनतम विनिर्देश ड्राफ्ट <https://tc39.es/ecma262/> पर है।
 
-To read about new bleeding-edge features, including those that are "almost standard" (so-called "stage 3"), see proposals at <https://github.com/tc39/proposals>.
+नई ब्लीडिंग-एज विशेषताओं के बारे में पढ़ने के लिए, जिनमें "लगभग मानक" (तथाकथित "स्टेज 3") शामिल हैं, <https://github.com/tc39/proposals> पर प्रस्ताव देखें।
 
-Also, if you're in developing for the browser, then there are other specs covered in the [second part](info:browser-environment) of the tutorial.
+इसके अलावा, यदि आप ब्राउज़र के लिए विकसित कर रहे हैं, तो ट्यूटोरियल के [दूसरे भाग] (जानकारी: ब्राउज़र-पर्यावरण) में शामिल अन्य चश्मा हैं।
```

Comment:
चश्मा is not the right word here. specs here refers to specifications lets use विशिष्टता